apps/web/screens/Home/Home.tsx

```diff
@@ -114,6 +114,7 @@ const Home: Screen<HomeProps> = ({ categories, news, page }) => {
           <NewLinks
             heading={n('newsTickerHeading')}
             seeMoreText={n('newsTickerSeeMore')}
+            showMoreButton
```

Comment:
This adds a button on the homepage:

![image](https://user-images.githubusercontent.com/43557895/154445708-4be83756-b96e-4ec2-854d-8f67f3085e53.png)

But shouldn't something happen if I click the button?

Reply:
Forgot that in there :)